configure

```diff
@@ -1272,10 +1292,9 @@ for ac_option in $@; do
 		_backend=`echo $ac_option | cut -d '=' -f 2`
 		;;
 	--enable-c++11)
-		_use_cxx11=yes
 		;;
-	--disable-c++11)
-		_use_cxx11=no
+	--disable-c++11|--force-c++98)
```

Comment:
I'm not sure I like the idea of adding an option to `configure` to support a C++ standard version that will likely no longer be accepted in the near future. 
What do you think about adding something like `--with-c++-dialect=c++98` instead?

Reply:
No sure it's better actually because then we need to add checks for the allowed version...

Reply:
I don’t have any strong preference either way. I guess the arguments for keeping it this way instead of using something like `--with-c++-dialect=` are that (1) it has a clearly identifiable end-of-useful-life where it can be removed, and (2) it doesn’t offer any easy avenue for users to attempt use of incompatible language standards.

Reply:
Ok, sounds good.

Reply:
Personally I'd prefer `--with-c++-dialect=foo` but I'm fine with either one.